pom.xml

```diff
@@ -32,17 +32,17 @@
   </modules>
 
   <properties>
-    <cobertura.version>2.0.3</cobertura.version>
-    <cobertura.maven.plugin.version>2.6</cobertura.maven.plugin.version>
+    <cobertura.version>2.1.1</cobertura.version>
+    <cobertura.maven.plugin.version>2.7</cobertura.maven.plugin.version>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
     <spring-platform.version>Brussels-SR5</spring-platform.version>
```

Comment:
There is a new Spring BOM release `Cairo-RELEASE` shall we move to this @massdosage ?

Reply:
I'd like to do that but I'd prefer holding off for this PR as I don't want to overcomplicate things and I worry that might introduce bigger changes. Also, spring platform as a "thing" is end-of-life and the recommendation from them is to replace this with the `spring-boot-dependencies` bom. I'll create a separate ticket for that in a bit.